kentikapi/client_integration_test.go

```diff
@@ -558,7 +559,7 @@ func TestClient_GetAgent(t *testing.T) {
 				assert.Equal(t, tt.expectedRequestID, r.GetId())
 			}
 
-			assert.Equal(t, tt.expectedResult.GetAgent().String(), response.GetAgent().String())
+			assert.True(t, proto.Equal(tt.expectedResult.GetAgent(), response.GetAgent()))
```

Comment:
```suggestion
			assert.True(t, proto.Equal(tt.expectedResult, result), fmt.Sprintf("Protobuf messages are not equal:\nexpected: %v\nactual:  %v", tt.expectedResult, result))
```
Notice: please rename response variable _result_

Reply:
Done